TECHNOLOGY_CHOICES.md

```diff
@@ -0,0 +1,52 @@
+# Technology Choices
+## Checks for determining database
+- Database should be able to store XML data
+- Database should be relational, because we need relations between files/pieces of files/models
+- Database must be provided with the solutions, for example part of a Docker image
+- Support must be available, in case of trouble.
+- Database must be open-source, because the whole solution must be open-source
+
+## PostgreSQL
+Homepage: https://www.postgresql.org/
+### Pros
+- Open Source, relational database
+- PostgreSQL does have XML functions for handling XML files (XMLTABLE)
+- XMLTABLE turns XML into a relational table format
+- Good support available
+- [Docker image](https://hub.docker.com/_/postgres) available
+
+### Cons
+- Not a native XML database
+
+## CouchDB
+Homepage: https://couchdb.apache.org/
+### Pros
+
+### Cons
+- Cannot store XML as key/property, but as blob. This makes querying a bit challanging.
+
+## eXist-db
+Homepage: http://exist-db.org/exist/apps/homepage/index.html
+### Pros
+- Native XML database
+- Fully Open Source
+- Cross-platform
+- Active community (weekly community call, Slack channels, books), so if we need support it's (almost) directly available
+- Docker image available
+- Multiple query languages like HTTP, REST, xQuery and xPath
+- Can act like a relational database, so relations between xml fragments for example is possible.
+- Includes XML validation
+
+### Cons
+- Using LGPL software is discouraged by Alliander when __modifying__ source code. So if we want to add features to eXist-db in the future, we might have a problem.
```

Comment:
we might have a problem can be changed into: the source needs to be released under the same license.

Reply:
Done